pkg/core/state/migrations/postgres/3_v0.24.0.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS providers
+(
+    source TEXT NOT NULL,
+    name TEXT NOT NULL,
+    version TEXT NOT NULL,
+    v_major INT NOT NULL,
+    v_minor INT NOT NULL,
+    v_patch INT NOT NULL,
+    v_pre TEXT NOT NULL,
+    v_meta TEXT NOT NULL,
+    tables JSONB NOT NULL, -- key: resource name, value: array of table names
+    signatures JSONB NOT NULL, -- key: resource name, value: table signature
+    CONSTRAINT "providers_id" PRIMARY KEY (source, name)
```

Comment:
nit: would create some index here for name, and version, although number will be quite low, maybe its an over optimization

Reply:
it's an over optimization as every source/name can have only one row.